src/locales/en.json

```diff
@@ -213,6 +213,7 @@
     "textNewLine": "Add new line (text)",
     "textFinish": "Finish editing (text)",
     "zoomToFit": "Zoom to fit all elements",
+    "zoomToFitSelection": "Zoom to fit selection",
```

Comment:
```suggestion
    "zoomToFitSelection": "Zoom to selection",
```

I would go with the most popular one

<img width="290" alt="Screenshot 2020-12-13 at 10 36 50 PM" src="https://user-images.githubusercontent.com/125676/102023100-d4867300-3d93-11eb-9b87-f3ccb429b78a.png">


<img width="315" alt="Screenshot 2020-12-13 at 10 36 58 PM" src="https://user-images.githubusercontent.com/125676/102023086-b882d180-3d93-11eb-9487-c406d295d5eb.png">

Reply:
Agreed as I stated above, "zoom to selection" is the standard + it's shorter :)